apollo-compiler/src/test/graphql/schema.sdl

```diff
@@ -311,8 +311,35 @@ type Query {
   starship(id: ID!): Starship
   tree: Tree
   typeWithGraphQLKeywords: TypeWithGraphQLKeywords
+
+  object(input: ReservedInput): ReservedObject
+  interface: ReservedInterface
+  union: ReservedUnion
+  scalar(scalar: ReservedScalar): ReservedScalar
+  enum(enum: ReservedEnum): ReservedEnum
+}
+
+type ReservedObject @targetName(name: "renamedObject") {
+  field: String
 }
 
+interface ReservedInterface @targetName(name: "renamedInterface") {
+  field: String
+}
+
+union ReservedUnion @targetName(name: "renamedUnion") = ReservedObject | Review
+
+input ReservedInput @targetName(name: "renamedInput") {
+  field: String
+}
+
+scalar ReservedScalar @targetName(name: "renamedScalar")
+
+enum ReservedEnum @targetName(name: "renamedEnum") {
+  VALUE
+}
```

Comment:
Mind adding this in a separate schema (in the target_name folder) so that it keeps this "fallback" schema reasonnably small (also avoids diffs in the `__schema` folder)?